### DISCUSSION TITLE
Need Info For Customizing Items Under General Options -> Profile

### DISCUSSION THREAD

**Discussion by j2onlineaccts** [rsjaffe/MIDI2LR#626] [Q&A] [answered]
I'm trying to understand how to customize the items under the profile tab in general options.  Including Modules, Tools Panels and Profiles.  Unlike the on options that are either a drop down list of actions/settings or a slider bar.  Everything under profile is a free for text field.

Thank you

**Comment by rsjaffe** [comment, marked as answer]
See https://github.com/rsjaffe/MIDI2LR/wiki/Profiles for instructions.

**Reply by j2onlineaccts** [reply]
Thank you, I'm about to test it but I understand the concept.  I just need to make sure the profile XML is located in the same location as my last used/default profile location?

**Reply by rsjaffe** [reply]
same directory
